src/main/java/com/epam/wl/entities/UserOrder.java

```diff
@@ -4,13 +4,13 @@
 import lombok.Data;
 
 @Data
-public class Order {
+public class UserOrder {
     private int id;
     private int editionId;
     private int userId;
     private OrderStatus status;
 
-    public Order(int id, int editionId, int userId, OrderStatus status) {
+    public UserOrder(int id, int editionId, int userId, OrderStatus status) {
```

Comment:
looks like this constructor is redundant

Reply:
Done